Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -1947,13 +1947,13 @@ FloatRect RenderObject::repaintRectInLocalCoordinates() const
 
 AffineTransform RenderObject::localTransform() const
 {
-    static const AffineTransform identity;
+    static constexpr AffineTransform identity;
     return identity;
```

Comment:
This function is not return a reference, so you could write `return AffineTransform();` instead.